fe/fe-core/src/main/java/com/starrocks/sql/optimizer/rule/implementation/StreamAggregateImplementationRule.java

```diff
@@ -0,0 +1,40 @@
+// This file is licensed under the Elastic License 2.0. Copyright 2021-present, StarRocks Inc.
+
```

Comment:
like `physical` operators, we add a separated directory?

Reply:
done